### PR TITLE
241 update (and patch fix)

### PR DESCRIPTION
**Pull request by jayfurmanek**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description

Update Serving to 2.4.1
also:
Fixes https://github.com/open-ce/tensorflow-serving-feedstock/issues/19

To address #19, this changes the A and B patches to be complete and separate, instead of B patching A. Patch A was already a patch of a patch, so with B *patching that* it was a bit mad.
Now, patch A applies when: 
 - cpu only
 - cuda10.2 and py38

And patch B applies when
 - cuda 10.2 and py<38
 - cuda 11.0 (any python)

This should be slightly more clean

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: tensorflow-serving-api/meta.yaml
Comment (npanpaliya) [inline tensorflow-serving-api/meta.yaml:3]: Should we not reset build number to 1?

File: tensorflow-serving/meta.yaml
Comment (npanpaliya) [inline tensorflow-serving/meta.yaml:15]: We would also want to update the selector while specifying requirements in meta.yaml.

File: tensorflow-serving-api/meta.yaml
Comment (jayfurmanek) [reply tensorflow-serving-api/meta.yaml:3]: Yes, will update

File: tensorflow-serving/meta.yaml
Comment (jayfurmanek) [reply tensorflow-serving/meta.yaml:15]: No changes should be needed for the reqs

**Comment by jayfurmanek** [comment]
@npanpaliya @bnemanich This one should be ready. I'll merge once approved.

**Review by npanpaliya — APPROVED** [review]

**Pull request merged**
